src/api/spec/controllers/webui/monitor_controller_spec.rb

```diff
@@ -19,6 +51,7 @@
 
   describe 'GET #update_building' do
     before do
+      stub_request(:get, "#{CONFIG['source_url']}/build/_workerstatus").and_return(body: xml_response)
```

Comment:
In line 59 is a comment that you could drop now -> "# it relays on a simulated VCR cassette"

Reply:
indeed :)